packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -217,6 +222,19 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
         alignItems: "center",
         justifyContent: "center"
     },
+    site_transparencyToggleButton: {
+        border: "none",
+        background: "none",
+        height: toPx(32),
+        width: toPx(32),
+        outline: "0",
+        verticalAlign: "middle"
+    },
+    site_transparencyToggleButtonIcon: {
+        display: "block",
+        verticalAlign: "middle",
+        fontSize: "0"
```

Comment:
do we need to define fontSize as "0"?

Reply:
It looks like this is using text-based styles to achieve vertical alignment, is that right? Is it possible to use flexbox instead? Flexbox seems to be more consistent across browsers.

Reply:
+1 on Nick's comment. On the opposite side of this we're using flexbox to accomplish the same thing. As a side note, I think the parent div here is floated, and we should avoid that since we can use flexbox there as well.